CONTRIBUTING.md

```diff
@@ -234,9 +234,7 @@ Examples:
 Before each PR follow these steps:
 
 1. Update the file in the [snippets folder](./snippets) (respects the alphabetical order).
-2. Update the [CHANGELOG.md](./CHANGELOG.md) file to reflect the changes.
-3. Add the info to the [snippets table](./README.md).
-4. Submit your PR.
+2. Submit your PR.
```

Comment:
@kellyhutchins should we keep the step 3 until we have a way to generate that table automatically? Otherwise 
there is no easy way for users to know what snippets they can use, right?

```suggestion
2. Add the info to the [snippets table](./README.md).
3. Submit your PR.
```

Reply:
Good point. I'll keep that one in place.